docs/admin/index.rst

```diff
@@ -0,0 +1,154 @@
+============================
+
+[TODO] Give some intro about Renku and a link
+
+This guide will help you deploying and customizing your own Renku instance in a cluster.
+
+A. Cluster requirements
+-----------------------
+
+To deploy Renku in a cluster, you need to have the following prerequisites:
+
+   - a `Kubernetes  <https://kubernetes.io/>`_ cluster
+       - `Setup Kubernetes on Openstack <prerequisites/k8s/openstack.html>`_
+       - You can also `Setup a Kubernetes cluser with Rancher <prerequisites/k8s/rancher.html>`_
+   - a `loadbalancer <prerequisites/loadbalancer.html>`_
+   - a `Gitlab runner <prerequisites/gitlabrunner.html>`_ VM for the docker image builder with **sufficient storage**
+   - a setup of `Helm <prerequisites/tiller.html>`_
+   - a way to obtain `TLS certificates <prerequisites/certificates.html>`_
+   - an `NGINX Ingress Controller  <prerequisites/nginx.html>`_ configured
+   - a `DNS <prerequisites/dns.html>`_ domain for Renku
+
+Optionally, you can choose to manage your own instances of the following:
+
+   - a `Gitlab stand alone instance <https://about.gitlab.com/install/>`_
+   - a Keycloak stand alone instance
+
+B. Pre-deployment steps
+-----------------------
+
+1. (optional) Stand alone configurations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   - `Stand alone Gitlab <configurations/standalone-gitlab.html>`_
+   - `Stand alone Keycloak <configurations/standalone-keycloak.html>`_
+
+2. Create and configure Renku PVs and PVCs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All of Renku information is stored in three volumes: Jena, Postgresql and Gitlab (if not stand alone).
+To this end, persistent volumes and persistent volume claims need to be created. You can use the following yaml files as a base to execute the `kubectl` commands below.
+
+   - `Renku PV yaml file <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/renku-pv.yaml>`_
+   - `Renku PVC yaml file <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/renku-pvc.yaml>`_
+
+   .. code-block:: console
+
+     $ kubectl create -f renku-pv.yaml
+     $ kubectl create ns renku
+     $ kubectl -n renku create -f renku-pvc.yaml
+     $ kubectl -n renku edit pvc renku-postgresql
+     $ kubectl get pv
+     $ kubectl get pvc -n renku
+     $ kubectl describe persistentvolumeclaim -n renku  ## this should not show any error, just PVs ready to be used
+
+Ensure that PVs have a `default storageclass` and the following value is set:
+
+   .. code-block:: console
+
+     metadata:
+       name: default
+       annotations:
+         storageclass.beta.kubernetes.io/is-default-class: "true"
+
+3. Create a renku-values.yaml file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This step is very important as it will define the values that helm will use to install your Renku instance.
+
+You can find a basic file in `Renku values file <https://github.com/SwissDataScienceCenter/renku-admin-docs/blob/master/renku-values.yaml>`_
```

Comment:
I wouldn't point them here - that file is not documented at all - just start with the relatively well-documented values file that is bundled with the helm chart in the renku repo? 

We should think about adapting the script from the dev-docs repo for this...

Reply:
I actually did document a bit better the one that's there , hence the pending PR.
About the script, exactly my thoughts, I guess we can do that for a second admin docs version.